swan-lake/learn/using-the-cli-tools.md

```diff
@@ -12,13 +12,13 @@ redirect_from:
   - /swan-lake/learn/using-the-cli-tools
 ---
 
-## Using the 'ballerina' Tool
+## Using the Ballerina Tool
 
-**Ballerina** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
+**`ballerina`** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
```

Comment:
This change isn't consistent with https://github.com/ballerina-platform/ballerina-dev-website/pull/1585/files#diff-13dab7a3e9ee8fe33dd736b9ed8c9ff722c937b0c7777610128b048dc83ccd38R260.

Reply:
Fixed.